dask_ml/compose/_column_transformer.py

```diff
@@ -185,7 +185,13 @@ def _hstack(self, Xs):
                 warnings.filterwarnings("ignore", "Concatenating", UserWarning)
                 return dd.concat(Xs, axis="columns")
         elif da.Array in types:
-            return da.hstack(Xs)
+            # To avoid changing dask core, this is the definition of the
+            # dask.array.hstack inlined:
```

Comment:
We're releasing fairly frequently these days.  If you're willing I encourage you to submit a small PR that passes `allow_unknown_chunksizes` through the `hstack` `vstack`, ... functions in dask.array.  That seems like it would benefit other applications as well.

Reply:
I created this PR to push this forward: https://github.com/dask/dask/pull/4287

Reply:
Thanks @vecchp.

I think we'll keep this compatibility code to allow us to pin our minimum bound at dask 1.0 for a while.

@ryan-deak-zefr could you update the comment to reflect that?

Reply:
@TomAugspurger: Please let me know if the current comment is sufficient.